src/exe/win32/console.cpp

```diff
@@ -38,9 +60,10 @@ void attachConsole() {
             } else {
                 // It takes a short time before the console becomes ready to be attached, unfortunately
                 // there is no API that would let us wait for it.
-                for (unsigned short i = 0; i < 20; i++) {
+                for (unsigned short i = 0; i < 40; i++) {
```

Comment:
**Question:** This will block the main thread, therefore it might be better to extrude this into a different thread?

Reply:
I implemented a more sophisticated (but potentially overcomplicated) version which I can push later.

I.a. it uses inter-process syncing via